observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -48,19 +46,19 @@ This update brings the following enhancements to Network Observability when IPse
 The following filtering options are now available for packets, flows, and metrics capture:
 
 * Track IPsec using `--enable_ipsec`
-* Value that determines the ratio of packets being sampled using `--sampling`
-* Filter flows using a custom query using `--query`
-* A comma separated list of interfaces to monitor using `--interfaces`
-* A comma separated list of interfaces to exclude using `--exclude_interfaces`
-* A comma separated list of metric names to generate using `--include_list`
+* Configure the ratio of packets being sampled by using `--sampling`
+* Filter flows using a custom query by using `--query`
+* Specify interfaces to monitor by using `--interfaces`
+* Specify interfaces to exclude by using `--exclude_interfaces`
+* Specify metric names to generate by using `--include_list`
```

Comment:
Would this be too much? Should add the periods, as these are full sentences now.
```suggestion
* Configure the ratio of packets being sampled by using the `--sampling` option.
* Filter flows using a custom query by using the `--query` option.
* Specify interfaces to monitor by using the `--interfaces` option.
* Specify interfaces to exclude by using the `--exclude_interfaces` option.
* Specify metric names to generate by using the `--include_list` option.
```